internal/pkg/cli/pipeline_status.go

```diff
@@ -36,6 +41,15 @@ func (o *pipelineStatusOpts) Ask() error {
 
 // Execute displays the status of the pipeline.
 func (o *pipelineStatusOpts) Execute() error {
+	err := o.initStatusDescriber(o)
+	if err != nil {
+		return fmt.Errorf("describe status of pipeline: %w", err)
+	}
+	pipelineStatus, err := o.statusDescriber.Describe()
+	if err != nil {
+		return fmt.Errorf("describe status of pipeline : %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("describe status of pipeline: %w", err)
```
There is an extra space here :P